content/docs/15.how-to-guides/sqlmesh.md

```diff
@@ -0,0 +1,195 @@
+---
+title: Using SQLMesh
+icon: /docs/icons/tutorial.svg
+stage: Getting Started
+topics:
+  - Integrations
+version: ">= 0.18.0"
+---
+
+Using SQLMesh to run dbt project via Kestra
+
+## Introduction to SQLMesh
+
+SQLMesh is an open source data transformation and modelling framework. It is a Python framework that automates everything needed to run a scalable data transformation platform. SQLMesh works with a variety of [engines and orchestrators](https://sqlmesh.readthedocs.io/en/stable/integrations/overview/).
+
+SQLMesh enables data teams to efficiently run and deploy data transformations written in SQL or Python.
+
+In this guide, we will learn how to run dbt project based on BigQuery using SQLMesh via Kestra.
+
+## SQLMesh based flow
+
+### Introduction to Flow
+
+Here is what we are going to do in the Kestra flow:
+
+1. Download the `orders.csv` using http download task.
+2. Create the table in BigQuery.
+3. Upload the data from the csv file into BigQuery table.
+4. Create dbt project which will create the BigQuery view from the BigQuery table.
+5. Create SQLMeshCLI task that will run the dbt project.
+
+### How will we implement dbt project?
+
+SQLMesh supports integration with a variety of tools like Airflow, dbt, dlt, etc. One of the common use-cases of SQLMesh is to run dbt projects.
+
+You can choose to pull your dbt project from Git repository as mentioned in the [how-to-guide on dbt](docs/how-to-guides/dbt) or create the namespace files for the complete project. Here, we will create the complete project using namespace files which we will create on the fly. You can later choose to push all the namespace files to GitHub repository using [PushNamespaceFiles](docs/how-to-guides/pushnamespacefiles).
```

Comment:
These links need to be relative going forward so they work in the Kestra in-app docs
```suggestion
You can choose to pull your dbt project from Git repository as mentioned in the [how-to-guide on dbt](docs/how-to-guides/dbt) or create the namespace files for the complete project. Here, we will create the complete project using namespace files which we will create on the fly. You can later choose to push all the namespace files to GitHub repository using [PushNamespaceFiles](docs/how-to-guides/pushnamespacefiles).
```

```suggestion
You can choose to pull your dbt project from Git repository as mentioned in the [How-to guide on dbt](./dbt.md) or create the namespace files for the complete project. Here, we will create the complete project using namespace files which we will create on the fly. You can later choose to push all the namespace files to GitHub repository using [PushNamespaceFiles](./pushnamespacefiles.md).
```